co2eq/meal.js

```diff
@@ -1,3 +1,6 @@
+import { carbonIntensity as purchaseCarbonIntensity } from './purchase';
+import { PURCHASE_CATEGORY_FOOD_RESTAURANT } from '../definitions';
+
 export const modelVersion = 1;
```

Comment:
Now that we have models that call other models we have to be careful.
Example:
if the purchase model is changed (to version 2) then the existing meals might not get updated because they are using the *meal* model (not the purchase model) at a version which didn't change.

Let's talk IRL.